packages/big-design/src/components/Dropdown/Dropdown.tsx

```diff
@@ -146,6 +147,7 @@ export const Dropdown = memo(
       (group: DropdownItemGroup) => {
         return (
           <>
+            {group.separated && <ListGroupSeparator />}
```

Comment:
One issue that we're going to run into using groupings is the space between the groups.  If you look at the designs for storefront manager ui, you'll notice that there isn't really any added space around the line separator.

Storefront UI designs:
<img width="204" alt="Screen Shot 2020-08-27 at 10 45 37 AM" src="https://user-images.githubusercontent.com/38708175/91477034-a9166180-e852-11ea-9e92-5f3df24f91bf.png">

Big Design dropdown w/o group labels:
<img width="256" alt="Screen Shot 2020-08-27 at 10 47 30 AM" src="https://user-images.githubusercontent.com/38708175/91477257-f561a180-e852-11ea-9655-201e286153a3.png">

Reply:
here's the Figmas for the new UI: https://www.figma.com/file/uFeZH9uVYkPxBcQgCr1ASC/MSF-Merchandising-concept?node-id=1994%3A65969 - Theme Actions -> Step 2 -> Channel Context

Reply:
I believe I fixed that gap in #443.

Reply:
Interesting - when I remove the label in the BD sandbox (https://developer.bigcommerce.com/big-design/dropdown), it still shows the gap. Having some screenshots on this PR would def help

Reply:
Yeah, we haven't release a new version/updated the docs since that change.